Source/interactivity_manager_impl.cpp

```diff
@@ -1229,18 +1246,23 @@ void interactivity_manager_impl::process_reply(const web::json::value& jsonReply
                 else
                 {
                     LOGS_INFO << L"Unhandled reply: " << jsonReply.serialize().c_str() << "to message " << message->to_string();
+                    messageWasProcessed = false;
                 }
             }
         }
         else
         {
             LOGS_INFO << "Unexpected json reply, no id";
+            messageWasProcessed = false;
         }
     }
     catch (std::exception e)
     {
         LOGS_ERROR << "Failed to parse service reply";
+        messageWasProcessed = false;

```

Comment:
Sounds good to me.

---
In reply to: [132999082](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132999082) [](ancestors = 132999082)